elements/dash-tooltip/demo.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  dash-tooltip {
+    margin-left: 8px;
+    padding: 4px;
+    background-color: #616161;
+    color: #fff;
+    border-radius: 3px;
+    opacity: 0;
+
+    /* Optionally, animate the tooltip. */
+    will-change: opacity;
```

Comment:
Suggest adding a note on the `will-change` property so devs know what it does
https://developer.mozilla.org/en-US/docs/Web/CSS/will-change